examples/example_neural_operator.py

```diff
@@ -0,0 +1,56 @@
+from dynabench.dataset import DynabenchIterator, download_equation
+from torch.utils.data import DataLoader
+from dynabench.model._grid._neural_operator import FourierNeuralOperator
+from dynabench.model.utils import PointIterativeWrapper
```

Comment:
A new IterativeWrapper for grid models needs to be created, which does not require the positions as input

Reply:
implemented grid wrapper in model/utils.py and used for neural operator -> points are no longer passed to forward()